### PR TITLE
Test

### PR DESCRIPTION
**Pull request by OleksandrFEdev**
[DEMO LINK](https://OleksandrFEdev.github.io/Museum_2/).

### CODE REVIEW

**Comment by TarasHoliuk** [comment]
1. Check the menu buttons location:
image
Screenshot from 2023-10-15 09-45-52

2. "Київ, вул. М. Грушевського, 6" should be a link (to Google Maps address, open it in the new tab)
image

3. In the mobile version red buttons should be full-width (see point 1 images)

4. Padding in this section looks too big (it should be the same in every section):
image

5. Image and button should be full-width in this section (check mockup):
image

6. Too big right padding:
image

7. "info@namu.kyiv.ua" should be a separate email link:
image

8. Anchor links don't work:
image

9. This button shouldn't be an anchor link. It should be a submit button (after clicking the input should be validated and cleared - see [the checklist](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md) from the previous landing, point 15). And read this checklist of general rules - apply the same here
image

10. Images are squeezed in the desktop version:
image

11. Add favicon

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
GJ :+1: 
Let's improve your landing a bit (read comments)

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,428 @@
...
+    type="image-favicon">
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="header__top--images">
+          <a href="#">
+            <img class="header__top--image"
+            src="/images/logo.png" alt="logo">
```
Comment (anastasiiavorobiova) [inline src/index.html:27]:
```suggestion
            <img 
              class="header__top--image"
              src="/images/logo.png" 
              alt="logo"
            >
```
Consider fixing indents according to the code style guide

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,428 @@
...
+          </a>
+
+          <div class="header__ua">
+            <a href="#menu">
+              <img class="header__top--burger"
+              src="/images/menu.png" alt="menu">
```
Comment (anastasiiavorobiova) [inline src/index.html:33]: The same issue here

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,462 @@
...
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Museum</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
```
Comment (anastasiiavorobiova) [inline src/index.html:10]:
```suggestion
    <link 
      rel="preconnect" 
      href="https://fonts.gstatic.com"
      crossorigin="anonymous"
    >
```

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,462 @@
...
+            <p class="header__ua--img">UA
+              <svg xmlns="http://www.w3.org/2000/svg"
+                width="10"
+                height="7"
+                viewBox="0 0 10 7"
+                fill="none">
+                <path fill-rule="evenodd"
+                  clip-rule="evenodd"
+                  d="M1.79691 1.08149C1.68826 0.972836 1.51209 0.972836 1.40343 1.08149L1.08149 1.40343C0.972836 1.51209 0.972836 1.68826 1.08149 1.79691L4.44592 5.16134C4.45639 5.17751 4.46872 5.19284 4.4829 5.20702L4.80484 5.52896C4.9135 5.63761 5.08966 5.63761 5.19832 5.52896L8.91851 1.80877C9.02716 1.70011 9.02716 1.52394 8.91851 1.41529L8.59657 1.09335C8.48791 0.984691 8.31174 0.984691 8.20309 1.09335L5.00593 4.29051L1.79691 1.08149Z"
+                  fill="#F1F5F4">
```
Comment (anastasiiavorobiova) [inline src/index.html:54]:
```suggestion
                  fill="#F1F5F4"
                >
```

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,462 @@
...
+
+    <aside class="menu page__menu" id="menu">
+
+      <div class="container">
+        <div class="menu__top">
+          <a href="#">
+            <img
+            class="menu__top--image"
+            src="/images/logo.png"
+            alt="logo"
```
Comment (anastasiiavorobiova) [inline src/index.html:88]:
```suggestion
              class="menu__top--image"
              src="/images/logo.png"
              alt="logo"
```

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,462 @@
...
+            >
+          </a>
+
+          <div class="menu__top--content">
+            <p class="menu__ua">UA
+              <svg xmlns="http://www.w3.org/2000/svg"
+              width="10"
+              height="7"
+              viewBox="0 0 10 7"
+              fill="none">
```
Comment (anastasiiavorobiova) [inline src/index.html:98]:
```suggestion
              <svg 
                xmlns="http://www.w3.org/2000/svg"
                width="10"
                height="7"
                viewBox="0 0 10 7"
                fill="none"
              >
```

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,462 @@
...
+                <path fill-rule="evenodd"
```
Comment (anastasiiavorobiova) [inline src/index.html:99]:
```suggestion
                <path 
                  fill-rule="evenodd"
```

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,462 @@
...
+                clip-rule="evenodd"
+                d="M1.79691 1.08149C1.68826 0.972836 1.51209 0.972836 1.40343 1.08149L1.08149 1.40343C0.972836 1.51209 0.972836 1.68826 1.08149 1.79691L4.44592 5.16134C4.45639 5.17751 4.46872 5.19284 4.4829 5.20702L4.80484 5.52896C4.9135 5.63761 5.08966 5.63761 5.19832 5.52896L8.91851 1.80877C9.02716 1.70011 9.02716 1.52394 8.91851 1.41529L8.59657 1.09335C8.48791 0.984691 8.31174 0.984691 8.20309 1.09335L5.00593 4.29051L1.79691 1.08149Z"
+                fill="#F1F5F4">
```
Comment (anastasiiavorobiova) [inline src/index.html:102]:
```suggestion
                fill="#F1F5F4"
              >
```

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,462 @@
...
+              </svg>
+            </p>
+
+            <a href="#home">
+              <img
+              src="/images/cross.png"
+              alt="close"
+              class="menu__closed"
+              >
```
Comment (anastasiiavorobiova) [inline src/index.html:111]:
```suggestion
              <img
                src="/images/cross.png"
                alt="close"
                class="menu__closed"
              >
```

File: src/index.html [hunk 10]
```diff
@@ -1,13 +1,462 @@
...
+          <div class="menu__navigation--line"></div>
+        </nav>
+        <a href="#" class="menu__navigation--btn">Купити квиток</a>
+
+        <div class="menu__bottom">
+          <div class="menu__bottom--img">
+            <img
+            src="/images/menu-image.png"
+            alt="Man"
+            class="menu__bottom--image"
```
Comment (anastasiiavorobiova) [inline src/index.html:148]:
```suggestion
              src="/images/menu-image.png"
              alt="Man"
              class="menu__bottom--image"
```

File: src/index.html [hunk 11]
```diff
@@ -1,13 +1,462 @@
...
+        <div class="main__content">
+          <h2 class="main__title">Актуальні виставки</h2>
+          <a href="#"
+          class="product__arhiv1">
+          Архів виставок
+            <img
+          src="/images/arrow.png"
+          alt="arrow"
+          class="product__arhiv--arrow"
+          >
```
Comment (anastasiiavorobiova) [inline src/index.html:167]:
```suggestion
          <a 
            href="#"
            class="product__arhiv1"
          >
            Архів виставок
            <img
              src="/images/arrow.png"
              alt="arrow"
              class="product__arhiv--arrow"
            >
```

File: src/index.html [hunk 12]
```diff
@@ -1,13 +1,462 @@
...
+                Купити квиток
+              </a>
+            </article>
+            <a href="#"
+            class="product__arhiv">
+            Архів виставок
+              <img src="/images/arrow.png"
+            alt="arrow"
+            class="product__arhiv--arrow"
+            >
```
Comment (anastasiiavorobiova) [inline src/index.html:197]:
```suggestion
            <a 
              href="#"
              class="product__arhiv"
            >
              Архів виставок
              <img 
                src="/images/arrow.png"
                alt="arrow"
                class="product__arhiv--arrow"
              >
```

File: src/index.html [hunk 13]
```diff
@@ -1,13 +1,462 @@
...
+          <div class="upcoming__events--container">
+            <h2 class="upcoming__events--top">Найближчі події</h2>
+            <a href="#"
+            class="upcoming__events--arhiv">
+            Календар подій
+              <img
+              src="/images/arrow.png"
+              alt="arrow"
+              class="upcoming__events--arrow"
+              >
```
Comment (anastasiiavorobiova) [inline src/index.html:212]:
```suggestion
            <a 
              href="#"
              class="upcoming__events--arhiv"
            >
            Календар подій
              <img
                src="/images/arrow.png"
                alt="arrow"
                class="upcoming__events--arrow"
              >
```

File: src/index.html [hunk 14]
```diff
@@ -1,13 +1,462 @@
...
+            </a>
+          </div>
+          <article class="upcoming__events--content">
+            <img
+            src="/images/main-man-photo.png"
+            alt="Man"
+            class="upcoming__events--image"
+            >
```
Comment (anastasiiavorobiova) [inline src/index.html:220]:
```suggestion
            <img
              src="/images/main-man-photo.png"
              alt="Man"
              class="upcoming__events--image"
            >
```

File: src/index.html [hunk 15]
```diff
@@ -1,13 +1,462 @@
...
+              </a>
+            </div>
+          </article>
+
+          <article class="upcoming__events--content">
+
+            <img
+            src="/images/main-girl-photo.png"
+            alt="Man"
+            class="upcoming__events--image"
```
Comment (anastasiiavorobiova) [inline src/index.html:236]:
```suggestion
              src="/images/main-girl-photo.png"
              alt="Man"
              class="upcoming__events--image"
```

File: src/index.html [hunk 16]
```diff
@@ -1,13 +1,462 @@
...
+              </a>
+            </div>
+
+            <a href="#"
+            class="upcoming__events--arhiv1">
+            Календар подій
+              <img
+            src="/images/arrow.png" alt="arrow"
+            class="upcoming__events--arrow"
+            >
```
Comment (anastasiiavorobiova) [inline src/index.html:253]:
```suggestion
            <a 
              href="#"
              class="upcoming__events--arhiv1"
            >
            Календар подій
              <img
                src="/images/arrow.png" 
                alt="arrow"
                class="upcoming__events--arrow"
              >
```

File: src/index.html [hunk 17]
```diff
@@ -1,13 +1,462 @@
...
+            <h2 class="news__top">Новини</h2>
+
+            <a href="#"
+            class="news__all--top">
+            Усі новини
+              <img
+            src="/images/arrow.png"
+            alt="arrow"
+            class="news__all--arrow"
+            >
```
Comment (anastasiiavorobiova) [inline src/index.html:282]:
```suggestion
            <a 
              href="#"
              class="news__all--top"
            >
              Усі новини
              <img
                src="/images/arrow.png"
                alt="arrow"
                class="news__all--arrow"
              >
```

File: src/index.html [hunk 18]
```diff
@@ -1,13 +1,462 @@
...
+            </a>
+          </div>
+          <div class="news__wrapper">
+            <article class="news__content">
+              <img
+              class="news__image"
+              src="/images/img_08.png"
+              alt="boock"
+              >
```
Comment (anastasiiavorobiova) [inline src/index.html:291]:
```suggestion
              <img
                class="news__image"
                src="/images/img_08.png"
                alt="boock"
              >
```

File: src/index.html [hunk 19]
```diff
@@ -1,13 +1,462 @@
...
+              <p class="news__data">9 серпня 2019</p>
+              <h3 class="news__title">Оголошення переможця</h3>
+              <p class="news__description">Друзі, сьогодні п'ятниця! А це означає, що час оголосити переможця розіграшу...</p>
+            </article>
+
+            <article class="news__content">
+              <img
+              class="news__image--second"
+              src="/images/img_09.png"
+              alt="cats"
```
Comment (anastasiiavorobiova) [inline src/index.html:301]:
```suggestion
                class="news__image--second"
                src="/images/img_09.png"
                alt="cats"
```

File: src/index.html [hunk 20]
```diff
@@ -1,13 +1,462 @@
...
+              <p class="news__description">Музей з левами не може просто так взяти і пропустити Міжнародний день котів!</p>
+
+              <a href="#"
+              class="news__all">
+              Усі новини
+                <img
+              src="/images/arrow.png"
+              alt="arrow"
+              class="news__all--arrow"
+              >
```
Comment (anastasiiavorobiova) [inline src/index.html:314]:
```suggestion
              <a 
                href="#"
                class="news__all"
              >
                Усі новини
                <img
                  src="/images/arrow.png"
                  alt="arrow"
                  class="news__all--arrow"
                >
```

File: src/index.html [hunk 21]
```diff
@@ -1,13 +1,462 @@
...
+            <h2 class="subscribe__title">Підпишіться на дайджест</h2>
+            <p class="subscribe__description">Першими дізнавайтесь про новини музею та розіграші, отримуйте запрошення на події та читайте статті від кураторів</p>
+          </div>
+          <div class="subscribe__center">
+            <form
+              action="https://namu.ua"
+              method="get"
+              class="subscription__form"
+              onsubmit="this.reset(); return false;"
+              >
```
Comment (anastasiiavorobiova) [inline src/index.html:333]:
```suggestion
            >
```

File: src/index.html [hunk 22]
```diff
@@ -1,13 +1,462 @@
...
+              <input
+                type="email"
+                name="email"
+                placeholder="e-mail"
+                class="subscribe__input"
+              >
+              <button
+              class="subscribe__button"
+              name="send__button"
+              type="submit"
```
Comment (anastasiiavorobiova) [inline src/index.html:343]:
```suggestion
                class="subscribe__button"
                name="send__button"
                type="submit"
```

File: src/index.html [hunk 23]
```diff
@@ -1,13 +1,462 @@
...
+              >
+              Підписатись
```
Comment (anastasiiavorobiova) [inline src/index.html:345]:
```suggestion
                Підписатись
```

File: src/index.html [hunk 24]
```diff
@@ -1,13 +1,462 @@
...
+              <a href="https://namu.ua" target="_blank">info@namu.kiev.ua</a>
+            </p>
+            <div class="footer__network">
+              <a href="https://www.facebook.com/namu.museum/?locale=uk_UA"
+                target="_blank">
+                <svg xmlns="http://www.w3.org/2000/svg"
+                width="20"
+                height="20"
+                viewBox="0 0 20 20"
+                fill="none">
```
Comment (anastasiiavorobiova) [inline src/index.html:369]:
```suggestion
              <a 
                href="https://www.facebook.com/namu.museum/?locale=uk_UA"
                target="_blank"
              >
                <svg 
                  xmlns="http://www.w3.org/2000/svg"
                  width="20"
                  height="20"
                  viewBox="0 0 20 20"
                  fill="none"
                >
```

File: src/index.html [hunk 25]
```diff
@@ -1,13 +1,462 @@
...
+                  <path d="M7.90909 10.9434V20H11.5455V10.9434H14.2727L15 7.16981H11.5455V4.5283C11.5455 3.96226 12.0909 3.39623 12.6364 3.39623H15V0H10.8182C9.90909 0 7.90909 1.32075 7.90909 3.39623V7.16981H5V10.9434H7.90909Z" fill="#F1F5F4">
+                </svg>
+              </a>
+              <a href="https://twitter.com/NAMU_120?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"
+                target="_blank">
+                <svg xmlns="http://www.w3.org/2000/svg"
+                width="20"
+                height="20"
+                viewBox="0 0 20 20"
+                fill="none">
```
Comment (anastasiiavorobiova) [inline src/index.html:379]:
```suggestion
              <a 
                href="https://twitter.com/NAMU_120?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"
                target="_blank"
              >
                <svg 
                  xmlns="http://www.w3.org/2000/svg"
                  width="20"
                  height="20"
                  viewBox="0 0 20 20"
                  fill="none"
                >
```

File: src/index.html [hunk 26]
```diff
@@ -1,13 +1,462 @@
...
+                  <path d="M3.78947 15.9669C2.94737 16.4548 0.912281 16.4412 0 16.3734C0.421053 16.78 3.57895 18 6.52632 18C9.47368 18 11.5789 17.5934 15.1579 14.3408C18.0211 11.7387 18.3158 7.971 18.1053 6.41244L20 4.37953H18.3158L19.3684 2.34663C18.5965 2.82095 17.0105 3.64763 16.8421 3.15973C16.6316 2.54986 13.6842 1.12682 11.1579 2.75314C9.13684 4.0542 9.47368 5.80257 9.68421 7.22553C5.89474 6.81902 1.05263 2.75314 1.05263 2.75314C-0.122503 4.86053 1.46337 7.41233 2.09911 8.43529L2.10526 8.4452L0.631579 8.03862C0.631579 9.82757 2.31579 11.0881 3.15789 11.6978H1.89474C2.56842 13.4867 4.70175 14.4761 5.68421 14.7471C5.40351 14.9504 4.63158 15.479 3.78947 15.9669Z" fill="#F1F5F4">
+                </svg>
+              </a>
+              <a href="https://t.me/s/namu_museum120"
+                target="_blank">
+                <svg xmlns="http://www.w3.org/2000/svg"
+                width="20"
+                height="20"
+                viewBox="0 0 20 20"
+                fill="none">
```
Comment (anastasiiavorobiova) [inline src/index.html:389]:
```suggestion
              <a 
                href="https://t.me/s/namu_museum120"
                target="_blank"
              >
                <svg 
                  xmlns="http://www.w3.org/2000/svg"
                  width="20"
                  height="20"
                  viewBox="0 0 20 20"
                  fill="none"
                >
```

File: src/index.html [hunk 27]
```diff
@@ -1,13 +1,462 @@
...
+                  d="M7.62887 17.4639L10.9278 14.2887L16.4948 19.0515L20 0L0 8.16495L5.56701 10.8866L7.62887 17.4639ZM8.0412 15.6495V12.0207L16.4948 3.85572L6.18553 10.433L8.0412 16.1031V15.6495Z" fill="#F1F5F4">
+                </svg>
+              </a>
+              <a href="https://www.instagram.com/namu.museum/"
+                target="_blank">
+                <svg xmlns="http://www.w3.org/2000/svg"
+                width="20"
+                height="20"
+                viewBox="0 0 20 20"
+                fill="none">
```
Comment (anastasiiavorobiova) [inline src/index.html:400]:
```suggestion
              <a 
                href="https://www.instagram.com/namu.museum/"
                target="_blank"
              >
                <svg 
                  xmlns="http://www.w3.org/2000/svg"
                  width="20"
                  height="20"
                  viewBox="0 0 20 20"
                  fill="none"
                >
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Check the previous review comments Screenshot 2023-10-22 at 15 17 37
Screenshot 2023-10-22 at 15 19 06
Screenshot 2023-10-22 at 15 19 12
Screenshot 2023-10-22 at 15 21 32
Screenshot 2023-10-22 at 15 21 40

**Comment by OleksandrFEdev** [comment]
[DEMO LINK](https://OleksandrFEdev.github.io/Museum_2/).

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Need to remove scroll when this menu is open
image

2. Add hover effects for all clickable elements on the page

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Add hover effects everywhere for clickable elements
image

2. Add cursor pointer here 
image

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Did you forget to redeploy the project?) Previous comments aren’t fixed

**Comment by OleksandrFEdev** [comment]
[DEMO LINK](https://OleksandrFEdev.github.io/Museum_2/).

**Review by lerastarynets — APPROVED** [review]
You've got this, keep pushing forward!

**Pull request closed without merging**